go.mod

```diff
@@ -5,32 +5,32 @@ require (
 	github.com/Masterminds/semver v0.0.0-20180403130225-3c92f33da7a8
 	github.com/Masterminds/vcs v1.13.1 // indirect
 	github.com/armon/go-radix v1.0.0 // indirect
-	github.com/beevik/etree v1.1.0
 	github.com/boltdb/bolt v1.3.1 // indirect
 	github.com/common-nighthawk/go-figure v0.0.0-20190529165535-67e0ed34491a
 	github.com/golang/dep v0.5.4
 	github.com/google/go-cmp v0.4.0 // indirect
-	github.com/jarcoal/httpmock v1.0.4
 	github.com/jedib0t/go-pretty/v6 v6.0.2
 	github.com/jmank88/nuts v0.3.0 // indirect
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
 	github.com/logrusorgru/aurora v0.0.0-20190803045625-94edacc10f9b
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/nightlyone/lockfile v0.0.0-20180618180623-0ad87eef1443 // indirect
-	github.com/package-url/packageurl-go v0.1.0
 	github.com/pelletier/go-toml v1.4.0 // indirect
-	github.com/recoilme/pudge v1.0.3
 	github.com/sdboyer/constext v0.0.0-20170321163424-836a14457353 // indirect
-	github.com/shopspring/decimal v0.0.0-20180709203117-cd690d0c9e24
-	github.com/sirupsen/logrus v1.5.0
+	github.com/shopspring/decimal v1.2.0
+	github.com/sirupsen/logrus v1.6.0
+	github.com/sonatype-nexus-community/go-sona-types v0.0.4
 	github.com/spf13/afero v1.2.2 // indirect
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.6.3
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.5.1
 	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a // indirect
-	golang.org/x/sys v0.0.0-20200519105757-fe76b779f299 // indirect
-	gopkg.in/go-playground/assert.v1 v1.2.1
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.3.0
 )
 
+replace github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
+
+replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200604202706-70a84ac30bf9
+
+replace golang.org/x/text => golang.org/x/text v0.3.3
+
```

Comment:
This whole block is to resolve vulns in upstream dependencies, leaving a comment for others to know why we did this.

Reply:
> This whole block is to resolve vulns in upstream dependencies, leaving a comment for others to know why we did this.

Excellent point. I will link this to Issue #152 Add notes about "how to fix vulnerable dependencies"

Would be a great chance to brag about how IQ Server Report can show you the next non-vulnerable version in many cases, etc.

db72bbd